easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -12,17 +12,34 @@ toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
 toolchainopts = {'usempi': True}
 
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
-sources = [SOURCELOWER_TAR_GZ]
+sources = ['vasp.5.4.1.24Jun15.tar.gz']
```

Comment:
@migueldiascosta shouldn't we stick to `SOURCELOWER_TAR_GZ` here, and (if we must) add a new easyconfig for the updated version that corresponds to `vasp.5.4.1.24Jun15.tar.gz`?